features/initialscreen.feature

```diff
@@ -0,0 +1,39 @@
+Feature : Initial Page
+As: an user
+I want to: see recommended content and the contents I already consume. And also being able to access other important pages.
+
+Scenario: Landing in the initial page
+Given: I am a user with login "ohbfinho" and password "oSportNaoSobeMais"
```

Comment:
só tira esses dois pontos : do Given:, And:, When: e Then:, porque assim a sintaxe tá errada